pandas/io/pytables.py

```diff
@@ -814,19 +806,19 @@ def select(
         Parameters
         ----------
         key : str
-                Object being retrieved from file.
+            Object being retrieved from file.
         where : list, default None
-                List of Term (or convertible) objects, optional.
+            List of Term (or convertible) objects, optional.
         start : int, default None
```

Comment:
int -> int or None

Reply:
Changed here and in the adjacent strings.